scraping/reddit/utils.py

```diff
@@ -37,6 +39,109 @@ def validate_reddit_content(
             content_size_bytes_validated=entity_to_validate.content_size_bytes,
         )
 
+    # Check Reddit id
+    if content_to_validate.id != actual_content.id:
+        bt.logging.info(
```

Comment:
Do we need to log this? I think we'll log the ValidationResult later (which includes the reason)

Reply:
We have been logging this historically in that it allows us to more easily see the diff. I am not sure if the validation result log contains the uri that failed for example?

Just before we logged if the content != content and now we repeat that for each check.